ai_genomics/utils/reading.py

```diff
@@ -0,0 +1,10 @@
+import json
+
+
+def read_json(data: str):
```

Comment:
```suggestion
def read_json(data: Union[pathlib.Path, str]) -> dict:
```

Reply:
Also a bit confusing that it takes `data` as an arg and returns `data`

Reply:
Blame GitHub copilot!